src/pages/GameCodePage/GameCodePage.tsx

```diff
@@ -22,27 +33,121 @@ const GameContainer = () => {
   }`;
   return (
     <>
-      <div className='w-[60rem]'>
+      <div className='w-[60rem] select-none'>
         <Highlight className='javascript'>{dummyCode}</Highlight>
       </div>
     </>
   );
 };
 const GameCodePage = () => {
+  const canvasRef = useCanvas({
+    setCanvas: (canvas: HTMLCanvasElement) => {
+      canvas.width = CANVAS_WIDTH;
+      canvas.height = CANVAS_HEIGHT;
+    },
+  });
+
+  interface ICarCoord {
+    x: number;
+    y: number;
+  }
```

Comment:
코드 컨벤션 때 접두사를 I_ 로 하기로 했던 것 같아서 다음과 같이 수정하면 어떨까용?! 😊
```suggestion
  interface I_CarCoord {
    x: number;
    y: number;
  }
```